inc/class-command.php

```diff
@@ -89,6 +91,9 @@ protected function execute( InputInterface $input, OutputInterface $output ) {
 			case 'status':
 				return $this->status( $input, $output );
 
+			case 'restart':
```

Comment:
Lets add an alias case for `reload` here too, just to be nice to folks more used to the vagrant commands.

Reply:
Okay, sure

Reply:
Done